code/api/src/modules/user/model.js

```diff
@@ -14,6 +14,12 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
+    },
+    primaryStyle: {
+      type: DataTypes.STRING
```

Comment:
I would like to know the difference between .TEXT and .STRING

Reply:
Per convo with @LHJE 
TEXT is for extremely long strings
STRING is a normal string.